packages/select-rich/test/lion-select-rich.test.js

```diff
@@ -204,6 +204,22 @@ describe('lion-select-rich', () => {
     expect(el.showsFeedbackFor.includes('error')).to.be.true;
   });
 
+  it('supports having no default selection initially', async () => {
+    const el = await fixture(html`
+      <lion-select-rich id="color" name="color" label="Favorite color" has-no-default-selected>
+        <lion-options slot="input">
+          <lion-option .choiceValue=${'red'}>Red</lion-option>
+          <lion-option .choiceValue=${'hotpink'} disabled>Hotpink</lion-option>
+          <lion-option .choiceValue=${'teal'}>Teal</lion-option>
+        </lion-options>
+      </lion-select-rich>
+    `);
+
+    console.log(el.selectedElement, el.modelValue);
```

Comment:
console..

Reply:
Having a bit of a monday-morning thing today 🙈

Reply:
I removed the console logs